### ISSUE TITLE
[1.16.5]org.cyclops.integrateddynamics.block.BlockCableConfig.<init>

### ISSUE THREAD

**Issue by ghost** [CyclopsMC/IntegratedDynamics#1094]
#### Issue type:

- :bug: Bug 

____

#### Short description:
SERVER DOWN

#### Steps to reproduce the problem:

1. install This mod
2. Boom.....

#### Expected behaviour:

____

#### Versions:

- This mod: IntegratedDynamics-1.16.5-1.10.0.jar
- Minecraft: 1.16.5
- Forge: forge-1.16.5-36.2.0

#### Log file:
https://paste.ubuntu.com/p/gDS3DHDwj2/

**Comment by rubensworks** [comment]
Thanks for reporting!

**Comment by rubensworks** [comment]
Duplicate of #1093. New release is pending on CurseForge.
